studio/src/store.js

```diff
@@ -7,6 +7,16 @@ import { reactiveStore } from './reactivity/reactive-store.js';
 const initialSearch = MasSearch.fromHash();
 const initialFilters = MasFilters.fromHash();
 
+const url = new URL(window.location.href);
+const pageParam = url.searchParams.get('page');
+
+const initialPage =
+    pageParam === 'welcome'
```

Comment:
in the absence of search query in the URL, the pageParam should be set to welcome so that the welcome page has always the same url.